infra/terraform/modules/data_access/teams/tests/test_create_team_bucket_policies.py

```diff
@@ -0,0 +1,32 @@
+import json
+from mock import call
+
+import pytest
+
+import teams
+
+from tests.conftest import TEST_TEAM_SLUG, TEST_READONLY_POLICY_DOCUMENT, \
+    TEST_READWRITE_POLICY_DOCUMENT, TEST_BUCKET_NAME
```

Comment:
Rather than using `\` to continue a line, wrap the list of import in parentheses, and maybe put one item per line, eg:
```python
from tests.conftest import (
    TEST_TEAM_SLUG,
    ...
)
```